wordpress_rock/files/etc/apache2/sites-available/000-default.conf

```diff
@@ -17,12 +17,8 @@
 	# modules, e.g.
 	#LogLevel info ssl:warn
 
-	# 2023/02/14
-	# Write logs to both files for loki log scraping and stdout for kubernetes
-	# container logs. Required for loki integration.
-	ErrorLog "|$ tee {APACHE_LOG_DIR}/error.log"
-	CustomLog "${APACHE_LOG_DIR}/access.log" combined
-	CustomLog /dev/stdout combined
+	CustomLog "| /usr/bin/rotatelogs -p /usr/bin/pruneaccesslogs ${APACHE_LOG_DIR}/access.%Y-%m-%d-%H-%M-%S.log 256M" logfmt
```

Comment:
We seem to be missing a `tee` here - is that deliberate? Might be good to include a comment explaining what we're doing here - we seem to have removed two lines of comments above.

Reply:
So, for the `CustomLog`, there is a statement below (`CustomLog /dev/stdout logfmt`) that will output the access log to stdout. And, for `ErrorLog`, using `rotatelogs` with the `-e` option will function like `tee` which also outputs the log to stdout. Added some comments to explain the logging statements here. Thanks!